Twig/CookieGuardExtension.php

```diff
@@ -87,4 +87,12 @@ private function getRequest()
 
         return $this->request;
     }
+
+    /**
+     * @return string
+     */
+    public function getName()
+    {
+        return get_called_class();
```

Comment:
Waarom doe je dit?

Deze functie zou je sowieso alleen moeten gebruiken binnen een static function:
> Gets the name of the class the static method is called in. 
https://secure.php.net/get_called_class

Reply:
Is aangepast naar get_class($this); zie https://github.com/twigphp/Twig/commit/71f03df01191053d4559bf4788b0c691346a6c40#diff-36bd39c2a5598ac745bda60d53706056